CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # The `zkvyper` changelog
 
-## [Unreleased]
+## [1.3.12 - 2023-10-26]
```

Comment:
The square bracket after the version please.

Reply:
Oops, resolved via 3fa7817842aec0802a3b7b3baf195888db82f9f2.